TramsDataApi/RequestModels/AcademyConversionProject/UpdateAcademyConversionProjectRequest.cs

```diff
@@ -1,8 +1,64 @@
-﻿namespace TramsDataApi.RequestModels.AcademyConversionProject
+﻿using System;
+
+namespace TramsDataApi.RequestModels.AcademyConversionProject
 {
     public class UpdateAcademyConversionProjectRequest
     {
+        public string ProjectStatus { get; set; }
+        public DateTime? ApplicationReceivedDate { get; set; }
+        public DateTime? AssignedDate { get; set; }
+        public DateTime? HeadTeacherBoardDate { get; set; }
+        public DateTime? OpeningDate { get; set; }
+        public DateTime? BaselineDate { get; set; }
+
+        //la summary page
+        public DateTime? LocalAuthorityInformationTemplateSentDate { get; set; }
+        public DateTime? LocalAuthorityInformationTemplateReturnedDate { get; set; }
+        public string LocalAuthorityInformationTemplateComments { get; set; }
+        public string LocalAuthorityInformationTemplateLink { get; set; }
+        public bool? LocalAuthorityInformationTemplateSectionComplete { get; set; }
+
+        //school/trust info
+        public string RecommendationForProject { get; set; }
+        public string Author { get; set; }
+        public string ClearedBy { get; set; }
+        public bool? IsAoRequired { get; set; }
```

Comment:
Should this be `AcademyOrderRequired` like the other model. Also in the other model its a `string`. I may have changed it to `bool?` though so could be wrong about that

Reply:
yeah it should be `AcademyOrderRequired` but I left it as I thought it could be sorted as part of school/trust info ticket, but on second thought I'll just sort it now so it's done!

and I changed it from bool to string in the db model to be consistent with the other yes/no fields such as `ViabilityIssues` and `FinancialDeficit` - as the existing fields in the sip db for the yes/no fields seem to be strings, so thought it'd be easier to do it this way so that we're not having to convert yes/no strings to bools when data is migrated to the new table